src/pages/Details.js

```diff
@@ -2,83 +2,82 @@ import React from 'react';
 import '../styling/Details.css'
 import Source from '../components/Source'
 import { dateToString } from '../util';
-import Carousel from 'react-bootstrap/Carousel' 
+import Carousel from 'react-bootstrap/Carousel'
 import logo from '../images/cwb-logo-reverse-w-tagline 1.png';
 import axios from 'axios';
 import { withRouter } from 'react-router-dom';
-
-
+ 
+ 
 class Details extends React.Component {
-   componentDidMount() {
-    const { match: { params } } = this.props;
-  
+  componentDidMount() {
+   const { match: { params } } = this.props;
     axios.get(`/details/${params.userId}`)
-      .then(({ data: user }) => {
-        console.log('user', user);
-  
+     .then(({ data: user }) => {
+       console.log('user', user);
         this.setState({ user });
-
-      });
-  }
-  //console.log({this.props.location.state})
-  //let currentSubmission;
-  //var currentSubmission = props.location.state.mySubmission;
-  //let var = this.props.location.state;
+ 
+     });
+ }
+ //console.log({this.props.location.state})
+ //let currentSubmission;
+ //var currentSubmission = props.location.state.mySubmission;
+ //let var = this.props.location.state;
+  render() {
+   console.log("hello");
+   console.log(this.props.location);
+   return (
+ 
+   <React.Fragment>
+      <Carousel>
+       <Carousel.Item>
+         <img src={this.props.location.submission.images[0]}  alt="logo"/>
+       </Carousel.Item>
+       <Carousel.Item>
+         <img src={logo}  alt="logo"/> 
```

Comment:
Change alts (message me for more info) but you will have to work with Karina to make the alts taking in info from the submission form